mappings/net/minecraft/data/client/model/BlockStateModelGenerator.mapping

```diff
@@ -2,8 +2,16 @@ CLASS net/minecraft/class_4910 net/minecraft/data/client/model/BlockStateModelGe
 	FIELD field_22830 blockStateCollector Ljava/util/function/Consumer;
 	FIELD field_22831 modelCollector Ljava/util/function/BiConsumer;
 	FIELD field_22832 simpleItemModelExemptionCollector Ljava/util/function/Consumer;
+	FIELD field_28548 CONNECTION_VARIANT_FUNCTIONS Ljava/util/Map;
+		COMMENT A map from a boolean property about connection on one direction to a
+		COMMENT function that creates a block state variant for connection on that
+		COMMENT direction with a given connection model.
+	FIELD field_28549 trapdoors Ljava/util/List;
```

Comment:
These are only trapdoors with no difference whatever the facing

Reply:
Yeah need to look into registerTrapdoor again